core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/ReferenceGenomeImpl.java

```diff
@@ -0,0 +1,46 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import uk.ac.bbsrc.tgac.miso.core.data.ReferenceGenome;
+
+@Entity
+@Table(name = "ReferenceGenome")
+public class ReferenceGenomeImpl implements ReferenceGenome {
+
+  @Id
+  @Column(unique = true, nullable = false)
```

Comment:
To support id auto generation in mysql change this line to:
@GeneratedValue(strategy = GenerationType.IDENTITY)